File: Prueba y error/ejercicios de semana santa java/one java/Library/Descuentos.java
package Library;

public class Descuentos {
    public static double calcularARL(double sueldo, int riesgo) {
        switch (riesgo) {
            case 1: return sueldo * 0.00522;
            case 2: return sueldo * 0.01044;
            case 3: return sueldo * 0.02436;
            case 4: return sueldo * 0.04350;
            case 5: return sueldo * 0.06960;
            default: return 0;
        }
    }
}

File: Prueba y error/ejercicios de semana santa php/One php/Library/Descuentos.php
<?php
namespace Library;

class Descuentos {
    public static function calcularARL(float $sueldoBase, int $riesgo): float {
        // Cálculo del ARL basado en el riesgo
        $porcentajeARL = [1 => 0.00522, 2 => 0.01044, 3 => 0.01980];
        return $sueldoBase * $porcentajeARL[$riesgo] ?? 0;
    }
}
